.github/workflows/build-test.yml

```diff
@@ -13,32 +13,32 @@ permissions:
   contents: read
 
 jobs:
-  build_image:
-    name: Build image
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
-        with:
-          persist-credentials: false
-
-      # The image is not pushed anywhere at the moment but serves the following purposes:
-      # (a) performs the npm run build when building the image, and
-      # (b) acts as a placeholder if we want to deploy as a container in the future.
-      - name: Build the image
-        uses: docker/build-push-action@263435318d21b8e681c14492fe198d362a7d2c83 # v6.18.0
-        with:
-          context: .
-          platforms: linux/amd64
-          push: false
-          build-args: |
-            HEAD_COMMIT_HASH=${{ github.sha }}
-          tags: |
-            ghcr.io/${{ github.repository_owner }}/bettergov:${{github.sha}}
-            ghcr.io/${{ github.repository_owner }}/bettergov:latest
+  # build_image:
+  #   name: Build image
```

Comment:
An alternative way is to add `if: false` here to prevent this job from running without needing to comment out this block.